baya/tests/settings.py

```diff
@@ -84,7 +84,9 @@
 
 TEST_RUNNER = 'django_nose.NoseTestSuiteRunner'
 
-MIDDLEWARE_CLASSES = (
+CSRF_USE_SESSIONS = False
```

Comment:
Can you elaborate on this change?

Reply:
That controls whether the csrf token is saved in the session. Tests would require additional mocking if enabled. 
 The default is `False` but just wanted to make it explicit that it's expected to be disabled. 

One of the Django 2.x failures was due to an admin view having CSRF validation hardcoded inside django. It can't be disabled since it's enabled by decorator.

https://docs.djangoproject.com/en/2.2/ref/settings/#csrf-use-sessions

Reply:
SGTM. Thanks for clarification.